toolkit/tools/internal/csvparser/csvparser.go

```diff
@@ -0,0 +1,78 @@
+package csvparser
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+var timeArray [][]string
+
+// Reads a CSV file and appends line by line to array
+func CSVToArray(filename string) {
+	file, err := os.Open(filename)
+
+	if err != nil {
+		fmt.Println("failed to open csv file")
+	}
+	defer file.Close()
+
+	fileScanner := bufio.NewScanner(file)
+
+	for fileScanner.Scan() {
+		timeArray = append(timeArray, strings.Split(fileScanner.Text(), ","))
+	}
+
+}
+
+// Return list of file paths
+func FilepathsToArray(parentDir string) []string {
+	// wd, _ := os.Getwd()
+	// idx := strings.Index(wd, "CBL-Mariner/toolkit") // 19 chars
+	// wd = wd[0 : idx+19]
+	// wd += "/tools/internal/timestamp/results/"
+
+	image_config_validator_path := parentDir + "/imageconfigvalidator.csv"
```

Comment:
An internal package should not be using hard-coded paths to files. This package may be moved to another tool, or we might move some of the folders around. The golang should be flexible enough to keep working. These types of paths need to be either passed in as a parameter, or calculated dynamically at runtime.